app/views/applications/show.html.erb

```diff
@@ -89,17 +98,26 @@
     </tbody>
   </table>
 <% else %>
-  <div class="alert alert-error">
-    <div>Couldn't get data from GitHub:</div>
-    <div><%= @github_error %></div>
+  <%= render "govuk_publishing_components/components/govspeak", {
+  } do %>
+    <div class="application-notice help-notice">
```

Comment:
Not sure where this appears, but is this the most suitable component for this?

Reply:
This is what it looks like:

<img width="987" alt="Screenshot 2019-12-02 at 11 42 29" src="https://user-images.githubusercontent.com/29889908/69956767-d79c2180-14f8-11ea-97a0-9085913180dd.png">

Happy for suggestions for alternatives